src/components/SideMenu/SideMenu.test.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  fireEvent,
+  getByTestId,
+  getByText,
+  render,
+  screen,
+} from "@testing-library/react";
+import AdminHomePage from "../../pages/AdminHomePage/AdminHomePage";
+import { MemoryRouter } from "react-router";
+import { routes } from "../../constants/routes";
+
+describe("Test side-menu", () => {
+  test("should render side menu on admin page", async () => {
+    render(
+      <MemoryRouter
+        initialEntries={[
+          {
+            pathname: routes.HOME,
+          },
+        ]}
+      >
+        <AdminHomePage />
+      </MemoryRouter>
+    );
+
+    const menuComponent = await screen.findByTestId(`menu-list`);
+
+    expect(menuComponent).toBeTruthy();
```

Comment:
```suggestion
    expect(menuComponent).toBeInTheDocument();
```
Use toBeInTheDocument instead of toBeTruthy for assertion, it's more up to date and more consistent with testing library api.